src/Cosmos/Protobuf/terra_wasm_v1beta1_tx.proto

```diff
@@ -15,5 +15,5 @@ message MsgExecuteContract {
     // ExecuteMsg json encoded message to be passed to the contract
     bytes execute_msg = 3;
     // Coins that are transferred to the contract on execution
-    repeated cosmos.base.v1beta1.Coin coins = 4;
+    repeated cosmos.base.v1beta1.Coin coins = 5;
```

Comment:
This change is not needed here
```suggestion
    repeated cosmos.base.v1beta1.Coin coins = 4;
```

Reply:
Hi @catenocrypt : Its needed. Wallet-core has a wrong serialization value. Kindly refer #2182

Reply:
Got it.  Could you add a comment note, so it will not get changed later (value 4 is missing but it should be that way).

Reply:
5